Source/Scene/DebugCameraPrimitive.js

```diff
@@ -130,14 +126,28 @@ define([
             var viewProjection = Matrix4.multiply(projection, view, scratchMatrix);
             var inverseViewProjection = Matrix4.inverse(viewProjection, scratchMatrix);
 
-            var positions = new Float64Array(8 * 3);
-            for (var i = 0; i < 8; ++i) {
-                var corner = Cartesian4.clone(frustumCornersNDC[i], scratchFrustumCorners[i]);
-                Matrix4.multiplyByVector(inverseViewProjection, corner, corner);
-                Cartesian3.divideByScalar(corner, corner.w, corner); // Handle the perspective divide
-                positions[i * 3] = corner.x;
-                positions[i * 3 + 1] = corner.y;
-                positions[i * 3 + 2] = corner.z;
+            var numFrustums = Math.max(1, Math.ceil(Math.log(frameState.far / frameState.near) / Math.log(frameState.farToNearRatio)));
```

Comment:
I think `FrameState` should store the exact splits rather than recalculating here.